docs/core/packages.md

```diff
@@ -133,7 +134,7 @@ The reverse would not be valid: targeting `netstandard1.6` with the 1.3.0 versio
 
 ### .NET Core application
 
-The .NET Core Application (TFM: `netcoreapp`) framework represents the packages and associated APIs that come with the .NET Core distribution and the console application model that it provides. .NET Core apps must use this framework, due to targeting the console application model, as should libraries that intended to run only on .NET Core. Using this framework restricts apps and libraries to running only on .NET Core. 
+The .NET Core Application ([Target Framework Moniker](../standard/frameworks.md): `netcoreapp`) framework represents the packages and associated APIs that come with the .NET Core distribution and the console application model that it provides. .NET Core apps must use this framework, due to targeting the console application model, as should libraries that intended to run only on .NET Core. Using this framework restricts apps and libraries to running only on .NET Core. 
```

Comment:
> The .NET Core ~~Application~~ ([Target Framework Moniker](../standard/frameworks.md): `netcoreapp`) 

Let's remove the Application suffix here -- `netcoreapp` is the TFM for .NET Core. The application part of the TFM is just an oddity that we shouldn't promote to the display text.